### PR TITLE
migrate: Fix existing down migrations

### PR DESCRIPTION
**Pull request by efritz**
This kept blowing up my dev environment.

### CODE REVIEW

File: migrations/1528395713_add_trigger_to_delete_orphan_repos.down.sql [hunk 1]
```diff
@@ -1,6 +1,5 @@
 BEGIN;
 
-DROP TRIGGER IF EXISTS trig_soft_delete_orphan_repos_for_external_service ON external_services;
-DROP FUNCTION IF EXISTS soft_delete_orphan_repos();
+DROP FUNCTION IF EXISTS soft_delete_orphan_repo_by_external_service_repos() CASCADE;
```
Comment (unknwon) [inline migrations/1528395713_add_trigger_to_delete_orphan_repos.down.sql:3]: TIL!

File: migrations/1528395711_add_soft_deleted_repository_name_func.up.sql [hunk 2]
```diff
@@ -1,6 +1,6 @@
 BEGIN;
 
-DROP FUNCTION IF EXISTS soft_deleted_repository_name();
+DROP FUNCTION IF EXISTS soft_deleted_repository_name(text);
```
Comment (unknwon) [inline migrations/1528395711_add_soft_deleted_repository_name_func.up.sql:3]: My bad 🤕, good catch!

**Review by unknwon — APPROVED** [review]
Thanks!

File: migrations/1528395711_add_soft_deleted_repository_name_func.up.sql (hunk 2 above)
Comment (unknwon) [reply migrations/1528395711_add_soft_deleted_repository_name_func.up.sql:3]: BTW, which PG version you're running on local dev? I often do multiple times of up/down on my dev machine, but I have 12 which seems to be "smarter" than old versions.

Comment (efritz) [reply migrations/1528395711_add_soft_deleted_repository_name_func.up.sql:3]: I'm also on 12.

**Pull request merged**